example/spec/testAPI/APISuccessSpec.js

```diff
@@ -181,5 +181,22 @@ describe('The Cumulus API', () => {
       expect(granuleRemoved).toEqual(true);
     });
   });
+
+  describe('logs endpoint', () => {
+    it('returns the execution logs', async () => {
+      const logs = await apiTestUtils.getLogs({ prefix: config.stackName });
+      expect(logs).not.toBe(undefined);
+      expect(logs.results.length).toEqual(10);
+    });
+
+    it('returns logs with taskName included', async () => {
```

Comment:
Can add tests to retrieve the logs based on the following?
1. workflow execution arn/name
2. step name, e. g. CmrStep